### ISSUE TITLE
Access Condition should display on archived item public page

### ISSUE THREAD

**Issue by pilasou** [DSpace/dspace-angular#2413]
On an item public page, if a file is under an temporary restriction or embargo, it is not possible for the user to know when this restriction/embargo will be waive.  

At the bitstream level, it is possible to set 4 different access options:  openaccess, administrator, embargo, lease (source : DSpace Rest contract [SubmissionAccessOptions Endpoints](https://github.com/DSpace/RestContract/blob/main/submissionaccessoptions.md)

**Improvement suggested**
The improvment deal only with temporary restriction (embargo) and is not taking into account the lease option or the administrator option for restricted items. 

The embargo end date is provided at the bitstream level either by the submitter or during the worklow. When an embargo is set during submission or workflow, the end date appears in the bitstream section as illustrated below :  
![embargo](https://github.com/DSpace/dspace-angular/assets/19291880/2317fad0-c3d9-4f2c-a591-90cee6e835cd)

The end date can be displayed the same way on the item page as show below: 
![item-pg-access-condition](https://github.com/DSpace/dspace-angular/assets/19291880/94792e02-eb60-41a5-ab1a-3986159e9967)

**Additional context**
May be similar to #1651

**Comment by tdonohue** [comment]
Agreed, this would be nice to have.  Needs a volunteer to implement.  Also needs analysis to determine if the embargo date is currently shared to the UI (it may not be).  If not, we may need to find a way to share it via the REST API.

**Comment by LincolnUniLTL** [comment]
This would be very useful for our institution too. Just to clarify that the label needs to remain associated with each bitstream, to account for cases where an item has multiple bitstreams with different access levels. For examples of our use cases, we have:

- items with a bitstream which is embargoed and will become Anon READ
- items with a bitstream which is embargoed and will only become available to a restricted group
- items with one bitstream which is embargoed and another which is open access
- items with one bitstream which is embargoed and another which is only available to a restricted group (in some cases these are our patrons, in other cases it's our Admin group)

...and so forth. We want it to be clear to users which files they have access to currently and/or may be able to access in future.
